config/transformer/models_list

```diff
@@ -20,4 +20,5 @@ openconfig-lacp-annot.yang
 ietf-ptp.yang
 ietf-ptp-annot.yang
 openconfig-system.yang
+openconfig-system-augments.yang
```

Comment:
The convention in the models/yang/ sub-directory is to use the -ext.yang suffix. Can you please check if this makes sense ?

Reply:
Done